config/hazelcast.yaml

```diff
@@ -37,5 +37,11 @@ hazelcast:
     hazelcast.invocation.max.retry.count: 20
     hazelcast.tcp.join.port.try.count: 30
     hazelcast.logging.type: log4j2
-    hazelcast.operation.generic.thread.count: 50
+    hazelcast.operation.generic.thread.count: 100
```

Comment:
why change the default value of thread count? The thread in operation thread pool never release. So if there is no need, don't increase it.

Reply:
roll back.